lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -33,7 +33,7 @@
 </script>
 
 <div class="batch-enrollment membership-section">
-  <h2> ${_("Batch Enrollment")} </h2>
+  <h3 class="hd hd-2">${_("Batch Enrollment")}</h3>
```

Comment:
Fair point. My goal here was to focus on functionality and markup, not styles, but I can see the value in updating the styles too.

A few months ago I brought over UXPL styles to handle these new heading classes, but scoped them a specific container. I might make another PR that opens them up and makes them available to a larger scope to cover these instances. Would you be OK with a second follow-up PR for that, or would you prefer I get them into this PR?